### ISSUE TITLE
OSC Support

### ISSUE THREAD

**Issue by icecaster** [raphaelradna/SpaceControl#4]
Would be great if SC supported OSC commands to allow integration with other software.

**Comment by raphaelradna** [comment]
How would you like to use OSC within the SC workflow?

**Comment by urbanpixellab** [comment]
Yes this would be nice, to get triggers and channel information in and out.
Like trigger playback of a file or move a sound spatial via osc. Then the position can be come from sensors in space.
But even without this it is a nice great software. In which language/framework is it written?

**Comment by raphaelradna** [comment]
But we don't control the position of a source in SC directly; rather, it moves along a user-defined path. I could still see some utility for a limited OSC implementation, for instance, to start/stop playback (there is only global playback, like a traditional DAW; not individual file triggers) or change levels.

I'm glad you're enjoying Space Control. It is built in Max/MSP, with some of the heavy lifting done in Javascript and C++ externals.

**Comment by icecaster** [comment]
Yes - basic OSC support: start, pause, stop would be good for triggering playback.

Changing individual parameters on the different channels would be helpful for interactive installations - further down the track
